docs/src/layouts/AppHeader.tsx

```diff
@@ -61,18 +67,54 @@ export default function AppHeader() {
           <HeaderNavBar />
         </Box>
         <Box sx={{ ml: 'auto' }} />
-        <Box sx={{ mr: { xs: 1, md: 2 } }}>
+        <Box sx={{ mr: { xs: 1, md: 1 } }}>
           <DeferredAppSearch />
         </Box>
-        <Box sx={{ display: { md: 'none' }, mr: 1 }}>
-          <HeaderNavDropdown />
-        </Box>
+        <Tooltip title={t('appFrame.github')} enterDelay={300}>
+          <IconButton
+            component="a"
+            color="inherit"
+            href={process.env.SOURCE_CODE_REPO}
+            data-ga-event-category="header"
+            data-ga-event-action="github"
+            sx={{
+              position: 'relative',
+              p: '6.5px',
+              mr: 1,
+              borderRadius: 1,
+              border: '1px solid',
+              bgcolor: (theme) =>
+                theme.palette.mode === 'dark' ? 'primaryDark.800' : 'transparent',
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark' ? 'primaryDark.500' : 'grey.200',
+              '& svg': { width: 18, height: 18 },
+              '&:focus': {
+                boxShadow: (theme) =>
+                  `0 0 0 1px ${
+                    theme.palette.mode === 'dark'
+                      ? theme.palette.primaryDark[600]
+                      : theme.palette.grey[200]
+                  }`,
+              },
+            }}
+          >
+            <GitHubIcon
+              fontSize="small"
+              sx={{
+                color: (theme) => (theme.palette.mode === 'dark' ? 'grey.100' : 'primary.main'),
```

Comment:
We can probably keep the color on the parent. I assume icon color is inherited.

Reply:
Done!